### ISSUE TITLE
I have an OG SCPH-1002  (BIOS v2.2 ) I can test

### ISSUE THREAD

**Issue by mgarcia-org** [brad-lin/FreePSXBoot#22]
Actually it's my SCEE Net Yaroze ;)

**Comment by brad-lin** [comment]
Images have been added for BIOS 2.0, 2.1, and 2.2. Could you try?

**Comment by mgarcia-org** [comment]
Thanks for that, I noticed it a few hours before your messaged here and already tried it out.
I formatted a working memory card and then flashed it with "freepsxboot-unirom-fastload-20210414-bios-2.2.mcd" via MemcardRex 1.8 using my DexDrive, it wrote the image fine and read it back fine.

The Net Yaroze thinks there's no memcard attached, ie it goes into the memcard/audio BIOS screen and in the memcard screen there's no memory card, I took the card out and plugged it back in, detects it, but continues to function like no card was plugged in.

I wanted try the ps1cardlink method of flashing the memcard, but the cardlink.exe PS-X didn't work on my modded 5502.

If there's something else I can try just let me know.
Cheers
Mike.

**Comment by mgarcia-org** [comment]
Hi,
I just tested your 'freepsxboot-unirom-fastload-20210419-bios-2.2'
I formatted the card first and loaded it up as above.
It goes into the memcard/audio screen, I clicked on the memcard, and it locks up.
I've left it for over a minute and tried it twice.

**Comment by brad-lin** [comment]
Hi,

2.2 has been confirmed working by other users. So it's either the Net Yaroze having a different BIOS than the SCPH-1002, or MemcardRex attempting to fix the memcard image before writing it. Could you try to read back the memory card after writing it, and check if there is any difference with the memory card image?

**Comment by fyatwyrio** [comment]
> Hi,
> 
> 2.2 has been confirmed working by other users. So it's either the Net Yaroze having a different BIOS than the SCPH-1002, or MemcardRex attempting to fix the memcard image before writing it. Could you try to read back the memory card after writing it, and check if there is any difference with the memory card image?

I also have been trying to use this with MemcardRex (1.9) and a DexDrive with no luck.  I wrote the freepsxboot-unirom-reduced-bios3.x.mcd to a memory card and then saved the card back out as a .MCD file (Bleem format in MemcardRex).  The files are very different although I don't know if that's from writing to the card or the writing out process.
BTW I have SCPH-5501 PU-18\1-664-537-62\September 1997 model.  Strangely Tonyhax via the Honda Castol Super bike save exploit is also not working.

**Comment by fyatwyrio** [comment]
Does anyone know of any utilities that could write the mcd image correctly on the playstation?  One that we could boot with the swap trick and then directly write the image?

**Comment by mgarcia-org** [comment]
Your right, MemcardRex doesn't work :/

I used PSXGameEdit v1.6 and worked first go!
It supports:
DexDrive
PlaySaver LPT
Caetla/Clones
XPlorer LPT

@fyatwyrio, I would guess it would be a nice addition to the Unirom ISO

Thanks and closed :)

**Comment by brad-lin** [comment]
Thanks for the feedback, I will update the readme with this information.

**Comment by hurda** [comment]
Does that mean the method with Memcarduino doesn't actually work?
Or is it just happening together with a Dexdrive?

**Comment by brad-lin** [comment]
Memcarduino works fine, as it allows to write raw data without attempting to modify anything. But memcarduino uses an arduino as the hardware, the dexdrive is completely different and memcarduino cannot be used with a dexdrive.

**Comment by hurda** [comment]
What I meant was:
MemcardRex + DexDrive = Bad
MemcardRex + Memcarduino = Good
PSXGameEdit + DexDrive = Good

Weird. :D

**Comment by fyatwyrio** [comment]
I tried using PSXGameEdit 1.6 with my DexDrive and it still didn't work.  I have been using a third party memory card though.  After selecting Memory Card from the playstation menu it just goes to the memory manager screen with nothing on the card.  I tried with an original Sony memory card and got further.  Now after selecting Memory Card the screen looks to blink once and sticks on a very pale blue screen.  I've let it go for a while at that screen and nothing happens.

I have a SCPH-5501 PU-18\1-664-537-62\September 1997 model playstation and have verified the Bios is 3.0 , checksum 8d8cb7e4, and a date of 11/18/96.

I've read just a few posts about some 5501's just not working with the exploit.   Has anyone found out any more information on why?

Thanks.
